dnf/rpm/transaction.py

```diff
@@ -111,9 +112,11 @@ def test(self, cb, conf={}):
         self.ts.setFlags(origflags)
 
         reserrors = []
-        if tserrors:
+        if tserrors is not None:
             for (descr, (etype, mount, need)) in tserrors:
                 reserrors.append(descr)
+            if not reserrors:
+                reserrors.append(_('Errors occurred during test transaction.'))
```

Comment:
This doesn't really tell the user any information about the error(s). Ideally we should provide the user with description of the error(s). It appears we don't have it here, but can we get it somehow? Lacking that, an error message saying "Unknown errors occured" is a pretty bad solution, but at least it provides the user with the information that we don't know anything about the errors.... (haha)

Reply:
The situation here was that rpm crashed completely and in this specific situation rpm doesn't provide any hint. We only known that something went wrong. The change was proposed by author of RPM and I trust him that this is the best and the only solution. There is a question whatever statement 'Errors occurred during test transaction.' is the best one. I am fine with the current statement.